sync-core/src/test/java/com/cloudant/sync/replication/AttachmentsPushTest.java

```diff
@@ -216,6 +216,48 @@ public void pushAttachmentsTest2() throws Exception {
 
     }
 
+    @Test
+    public void pushAttachmentsStubsCorrectlySent() throws Exception {
+        // more complex test with attachments changing between revisions
+        String attachmentName1 = "attachment_1.txt";
+        String attachmentName2 = "attachment_2.txt";
+        populateSomeDataInLocalDatastore();
```

Comment:
Why not just create a single document rather than calling this slightly indirect method of doing so?